src/Microsoft.ML.Transforms/Text/WordTokenizing.cs

```diff
@@ -410,8 +426,10 @@ public sealed class WordTokenizingEstimator : TrivialEstimator<WordTokenizingTra
         /// Tokenize incoming text in <paramref name="inputColumnName"/> and output the tokens as <paramref name="outputColumnName"/>.
         /// </summary>
         /// <param name="env">The environment.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// The output column is of type variable vector of string.</param>
```

Comment:
>string [](start = 60, length = 6)

`text` here and below

Reply:
Not public member I will not fix it for now.

---
In reply to: [276870603](https://github.com/dotnet/machinelearning/pull/3418#discussion_r276870603) [](ancestors = 276870603)